scout/server/blueprints/variants/templates/variants/utils.html

```diff
@@ -1,6 +1,17 @@
 {% import "bootstrap/wtf.html" as wtf %}
 {% from "utils.html" import comments_table %}
 
+{% macro mark_heteroplasmic_mt(individuals, samples) %}
+  {% for ind in individuals if ind.phenotype == 2 %}
+    {% for gt in samples|selectattr("sample_id", "equalto", ind.individual_id) %}
```

Comment:
It works nicely:
![Uploading image.png…]()

Perhaps make the badge a bit smaller so it fits better the column? (tested with chrome)

Reply:
If the badge looks OK on your browser is fine, could be the laptop I'm using today!

Reply:
I made the badge smaller by removing the `Het:` prefix to make it look good on smaller screens.